data/PA/incentives.json

```diff
@@ -0,0 +1,1023 @@
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-30",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "percent",
+      "number": 0.75,
+      "maximum": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Rebate for 75% of costs up to $500 for attic insulation in an electric heated house."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-31",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "percent",
+      "number": 0.75,
+      "maximum": 200
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Rebate for 75% of costs up to $200 for attic insulation (central AC, non-electric heat)."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-32",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "percent",
+      "number": 0.75,
+      "maximum": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Rebate for 75% of costs up to $500 for basement wall insulation in an electric heated house."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-33",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "percent",
+      "number": 0.75,
+      "maximum": 200
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Rebate for 75% of costs up to $200 for basement wall insulation (central AC, non-electric heat)."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-34",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "dollars_per_unit",
+      "number": 0.25,
+      "unit": "cmf50",
+      "maximum": 200
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$200 rebate for air sealing. Air infiltration reduction (@ CFM50) x $0.25 up to a max of $200."
+    },
```

Comment:
Now that I'm looking at it, this is going to be nonsense to a layperson. It will show in the frontend as "$0.25/CFM50 up to $200 for weatherization".

Maybe we drop the per-unit part, code it as `maximum: 200` and only talk about CFM50 in the description?